### PR TITLE
[FIX] fix the crashing of app when corrupted files are loaded

### PR DESCRIPTION
**Pull request by ghost**
The respective issue can be found at #1286 
Crashing of app is fixed and added the proper pop-up message.

### CODE REVIEW

File: src/core/libmaven/mzSample.cpp [hunk 1]
```diff
@@ -810,8 +810,7 @@ void mzSample::parseMzXML(const char* filename)
         setInstrumentSettigs(doc, spectrumstore);
         // parse mzXML information from the scan
         parseMzXMLData(spectrumstore);
-    } else
-        throw MavenException(ErrorMsg::ParsemzXml);
```
Comment (sakshikukreja14) [inline src/core/libmaven/mzSample.cpp:814]: @MdAzmatAli  why is this exception removed??? Wasn't it getting handled somewhere? I see this exception is thrown in other such functions too. This means those would also lead to a crash if the conditions match.

File: src/gui/mzroll/mainwindow.cpp [hunk 2]
```diff
@@ -683,7 +683,11 @@ using namespace mzUtils;
                     msgBox.setInformativeText("Please make sure your file "
                                               "import settings do not conflict "
                                               "with the type of samples being "
-                                              "loaded.");
+                                              "loaded.\n\nIf your settings are "
```
Comment (sakshikukreja14) [inline src/gui/mzroll/mainwindow.cpp]: your file import* settings

File: src/core/libmaven/mzSample.cpp (hunk 1 above)
Comment (saif-el) [reply src/core/libmaven/mzSample.cpp:814]: @MdAzmatAli Sakshi is right. These exceptions are thrown in a few other places in mzSample.cpp. They should ideally be caught and informed about the user on what we were expecting and what went wrong.

For example, in mzfileio.cpp, we [catch memory allocation exception](https://github.com/ElucidataInc/ElMaven/blob/develop/src/gui/mzroll/mzfileio.cpp#L82-L88) when user runs out of RAM while loading samples. Something similar should happen in this case as well - catch the specific type of exception (`ErrorMsg::ParsemzXml`) and then tell user that something is wrong with the file.

**Comment by saif-el** [comment]
And why is CI complaining about a C++17 exception. There were no changes related to anything like that.

**Pull request closed without merging**